ci/install-cxx.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+# We use https://cxx.rs to generate C++ and Rust bridge code.  If you change
+# rust/src/lib.rs, you will need to install the tool.
+set -xeuo pipefail
+CXX_VER=$(cargo metadata --format-version 1 | jq -r '.packages[]|select(.name == "cxx").version')
+mkdir -p target
+time cargo install --root=target/cxxbridge cxxbridge-cmd --version "${CXX_VER}"
```

Comment:
This currently duplicates `ci/installdeps.sh`.

Thinking more on this, I think it makes sense to have `ci/installdeps.sh` be a superset which includes installing cxx tooling, but maybe let's just have it execute this script like we do in `ci/verify-cxx.sh` to avoid duplication? (Also the comment there needs an update.)

Reply:
The way I think of it, `ci/installdeps.sh` is basically "stuff we need to build a source archive" (including vendor) to replicate more precisely the Koji-style offline build model.

So it shouldn't install cxx.

Reply:
Sure, WFM!